### PR TITLE
Test the persistence as part of end to end testing

### PR DESCRIPTION
**Pull request by erbridge**
The tests are failing due to a missing `property.id` object (possibly explained by parameter filtering on the backend), and an image not persisted for `property.otherComments.images` (no ideas currently).

*Update*: Those were both actual bugs that are also fixed here.

### CODE REVIEW

**Comment by erbridge** [comment]
If you have any thoughts on the latter issue @LBHELewis, that would be helpful.

File: components/makeSubmit.tsx [hunk 1]
```diff
@@ -17,33 +17,47 @@ export const makeSubmit = (
...
+                i > 0
+                  ? "submit-button lbh-button--secondary govuk-button--secondary"
+                  : "submit-button"
+              }
+              onClick={async (): Promise<void> => {
+                try {
+                  await onSubmit();
+                } catch (error) {
+                  // This is invisible to the user, so we should do something to
+                  // display it to them.
```
Comment (LBHELewis) [inline components/makeSubmit.tsx]: Has this been captured somewhere?

File: helpers/imageToBase64.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    };
+
+    reader.onerror = (error): void => {
+      reject(error);
+    };
+
+    reader.readAsDataURL(file);
+  });
+
+export default imageToBase64;
```
Comment (LBHELewis) [inline helpers/imageToBase64.ts]: Nice

File: layouts/MainLayout.tsx [hunk 3]
```diff
@@ -37,204 +37,203 @@ const MainLayout = ({
...
+          rel="manifest"
+          href={`${process.env.BASE_URL}/manifest.webmanifest`}
+        />
+        <link
+          rel="shortcut icon"
+          href={`${process.env.BASE_URL}/favicons/favicon.ico`}
+        />
+        <link
+          rel="manifest"
+          href={`${process.env.BASE_URL}/favicons/manifest.json`}
```
Comment (LBHELewis) [inline layouts/MainLayout.tsx]: I think this should be removed?

**Review by LBHELewis — CHANGES_REQUESTED** [review]
Looks good - just the manifest issue to fix, then good to merge

File: components/makeSubmit.tsx (hunk 1 above)
Comment (erbridge) [reply components/makeSubmit.tsx]: There's a backlog task for making sure we handle errors well generally.

File: layouts/MainLayout.tsx (hunk 3 above)
Comment (erbridge) [reply layouts/MainLayout.tsx]: Oops. Yes.

**Pull request merged**